src/util.h

```diff
@@ -75,29 +75,29 @@ class CClient;  // forward declaration of CClient
 
 /* Global functions ***********************************************************/
 // converting double to short
-inline short Double2Short ( const double dInput )
+static inline float clipFloat ( const float dInput )
```

Comment:
Interesting remark. What about the speed of the function qBound()?

Reply:
It's inlined in QtCore/global.h, so it's no worse than the existing code.

Reply:
Ok, so than we should keep the "ClipFloat" function but inside that function we should use the qBound function as you have proposed (keeping the ClipFlot to avoid that we have to specify the -1 to 1 on all places again and again). @hselasky Can you please make this change?

Reply:
Fixed.